include/ofi_mem.h

```diff
@@ -358,20 +357,69 @@ static inline void *util_buf_get(struct util_buf_pool *pool)
 	struct util_buf_footer *buf_ftr;
 
 	assert(!pool->attr.indexing.ordered);
-
-	slist_remove_head_container(&pool->list.buffers, struct util_buf_footer,
-				    buf_ftr, entry.slist);
-	assert(++buf_ftr->region->num_used);
+	dlist_pop_front(&pool->list.buffers, struct util_buf_footer, 
```

Comment:
`util_buf_gc_get`
and 
`util_buf_get`

Reply:
Created `util_buf_gc_get`. Done

Reply:
`dlist_pop_front(&pool->list.buffers.dlist, struct util_buf_footer, ... `